string item_name
int64 item_id
int64 quantity

string item_type
int64 feducial_id
